packages/components/src/FormField/FormField.tsx

```diff
@@ -12,6 +12,11 @@ export interface FormFieldProps {
    */
   readonly align?: "center" | "right";
 
+  /**
+   * Determines if browser form autocomplete is enabled.
+   */
+  readonly autocomplete?: string;
```

Comment:
I think a `boolean` would make more sense here.

Reply:
This one caught me out too. Autocomplete as used by browsers isn't a boolean. You can actually specify types to autocomplete on. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
The value section is extensive. If we used a boolean we would have to convert it to a string of password to turn it off for all browsers. I think leaving it as a string is better.